action-scripts/brownie/scripts/script_utils.py

```diff
@@ -121,16 +124,34 @@ def format_into_report(file: dict, transactions: list[dict]) -> str:
     return file_report
 
 
+def prettify_contract_inputs_values(chain, contracts_inputs_values):
+    addr = AddrBook(chain)
+    perm = BalPermissions(chain)
+    outputs = {}
+    for k,v in contracts_inputs_values.items():
```

Comment:
Whole logic should be commented since it's not clear what's going on in this function at all